### PR TITLE
updates the usage.md file with info on samplesheet

### PR DESCRIPTION
**Pull request by jemten**
Lane is still part of the samplesheet but we should probably consider switching to something else
#126 
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/raredisease/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/raredisease _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR> -stub`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -10,45 +10,39 @@
...
-TREATMENT_REP3,AEG588A6_S6_L003_R1_001.fastq.gz,
-TREATMENT_REP3,AEG588A6_S6_L004_R1_001.fastq.gz,
+sample,lane,fastq_1,fastq_2,gender,phenotype,paternal_id,maternal_id,case_id
+AEG588A1,2,AEG588A1_S1_L002_R1_001.fastq.gz,AEG588A1_S1_L002_R2_001.fastq.gz,1,2,AEG588A3,AEG588A2,fam_1
+AEG588A1,3,AEG588A1_S1_L003_R1_001.fastq.gz,AEG588A1_S1_L003_R2_001.fastq.gz,1,2,AEG588A3,AEG588A2,fam_1
+AEG588A2,4,AEG588A2_S1_L004_R1_001.fastq.gz,AEG588A2_S1_L004_R2_001.fastq.gz,2,1,,,fam_1
+AEG588A3,4,AEG588A3_S1_L004_R1_001.fastq.gz,AEG588A3_S1_L004_R2_001.fastq.gz,1,1,,,fam_1
 ```
 
+The pipeline will auto-detect whether a sample is single- or paired-end using the information provided in the samplesheet. The pedigree information (sex/gender and phenotype) should be provided using the ped file format way of annotating.
```
Comment (ramprasadn) [inline docs/usage.md:33]: The pedigree information in the samplesheet (sex/gender and phenotype) should be provided as they would be for a ped file (ex, 1 for male, 2 for female, other for unknown).

Comment (projectoriented) [reply docs/usage.md:33]: It looks like it's described here: https://github.com/nf-core/raredisease/pull/135/files#diff-72376d0b487d7231cf31959bf266f6aea098e899743bae02e36d176cef4db476R41

**Review by projectoriented — APPROVED** [review]
👍

File: docs/usage.md (hunk 1 above)
Comment (jemten) [reply docs/usage.md:33]: Good suggestion! I'll change that sentence

**Pull request merged**